ArduPlane/Parameters.h

```diff
@@ -347,6 +347,9 @@ class Parameters {
         k_param_mixing_offset,
         k_param_dspoiler_rud_rate,
 
+        // 250: ArduSoar parameters
+        k_param_soaring_controller,
```

Comment:
Unless soaring is deeply nesting other groups of parameters it might make more sense to stick it within the G2 set, rather then burning one of the last upper level indexs. From a quick scan it doesn't look like it nests additional parameters, so if it went into G2 it would be fine.